Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This Dockerfile builds a mainnet version for API3 DAO dashboard
+
+FROM node:lts-alpine as builder
+RUN apk add --no-cache git
+WORKDIR /usr/src/app
+ADD . .
+COPY src/contract-deployments/mainnet-dao.json ./src/contract-deployments/localhost-dao.example.json
+RUN yarn 
+RUN yarn tsc && yarn build
+
+FROM nginx:alpine
```

Comment:
Do we need nginx for this? Can't we just expose port 3000 from the create-react-app server? It's wouldn't be using the output html/js files, but I don't think that's a problem for this?

Reply:
`create-react-app` is for development purposes only, `webpack` that is under the hood of it is not designed to serve files efficiently, and `nginx` is the simplest way to serve static files when you have them packaged with as little resources as possible.

So it matters in terms of size of the running container, and the memory consumption, this is why builder and multi-stage patterns are used in docker files